### PR TITLE
Fix sitemap task

### PR DESCRIPTION
**Pull request by tavriaforever**
@tormozz48

### CODE REVIEW

File: src/tasks/sitemap-xml.js [hunk 1]
```diff
@@ -53,8 +57,13 @@ module.exports = function (target) {
...
         })
         .then(function (records) {
+            var urls = {
+                '@': { xmlns: 'http://www.sitemaps.org/schemas/sitemap/0.9' },
+                url: records
+            };
+
             // convert json model to xml format
-            return levelDb.get().put('sitemapXml', js2xml('urlset', { url: records }));
+            return levelDb.get().put('sitemapXml', js2xml('urlset', urls));
```
Comment (tormozz48) [inline src/tasks/sitemap-xml.js]: Need to save json model too

**Comment by tavriaforever** [comment]
update

**Comment by tormozz48** [comment]
:ok:

**Comment by tormozz48** [comment]
Remember that you should release new tag vx.y.z with updated x.y.z version number in package.json file

**Comment by tormozz48** [comment]
Also you should publish this package into npm. I gave you all rights for this

**Comment by tavriaforever** [comment]
This is a minor version isn't it?

**Comment by tormozz48** [comment]
Yes

**Pull request merged**